releasenotes/notes/apm-cpu-warn-4930a3d52fe06014.yaml

```
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    The APM Agent now logs a warning when it's falling behind.
```

```suggestion
    APM: A warning is now logged when the agent is under heavy load.
```
We have to prefix all trace-agent entries with `APM`. Perhaps this needs to be documented somewhere. Also, I'd rephrase. "Falling behind" is ambiguous.